src/bp/core/module-loader.ts

```diff
@@ -159,6 +163,11 @@ export class ModuleLoader {
       await resourceLoader.enableResources()
       await resourceLoader.runMigrations()
       await resourceLoader.importResources()
+
+      const migrationDirs = await resourceLoader.getDbMigrationsDir()
```

Comment:
I know I originally put migrations in ResourceLoader but I think I was highly intoxicated as it makes no sense; migrations aren't resources.

We already export modules path to `process.LOADED_MODULES`, I think it would be best to just do it in the MigrationService..

Reply:
Good point. I'll change that